scripts/build-bundle

```diff
@@ -11,13 +11,63 @@ cd ${TOP_DIR}
 
 addons_path=../addons
 if [ ! -d ${addons_path} ];then
-    echo "No existing addons source. Pulling..."
-    git clone --branch main --single-branch --depth 1 https://github.com/harvester/addons.git ../addons
+  echo "No existing addons source. Pulling..."
+  git clone --branch main --single-branch --depth 1 https://github.com/harvester/addons.git ../addons
 fi
 
+source ${SCRIPTS_DIR}/version-monitoring
+source ${SCRIPTS_DIR}/version-logging
+
+update_monitoring_logging_version() {
```

Comment:
do we still need this change? The addons PR is now merged, and ideally the addons should be source of truth for all addon related info.

Reply:
@ibrokethecloud Please check this comment https://github.com/harvester/harvester-installer/pull/792#issuecomment-2275130949, I need more time to finally finish it in the possible next PR.

Reply:
@w13915984028 Do you want to complete those changes in this PR? I guess we still have time before 1.4.0.

Reply:
@bk201 
I have a bunch of tasks in parallel, lets merge the PR and I will return to this topic after others are done. thanks.

Reply:
thanks, can we track the enhancement with a ticket?

Reply:
The ticket is alreay there, I added this to the task lists 

https://github.com/harvester/harvester/issues/6289#issue-2453393645